daemon/daemon_no_aufs.go

```diff
@@ -1,4 +1,4 @@
-// +build exclude_graphdriver_aufs,linux
+// +build exclude_graphdriver_aufs,linux !linux
```

Comment:
It doesn't exist on Windows :). I -think- you need to keep the original +build exclude_graphdriver_aufs,linux, but the second part depends on your intent - should this be included in FreeBSD?